### PR TITLE
WD-12825 - rebrand /azure/fips

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand /azure/fips. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14144.demos.haus/azure/fips)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-Azure-bubble?node-id=1-2233&t=5AOamUxpwm8vWNig-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12825](https://warthogs.atlassian.net/browse/WD-12825)

[WD-12825]: https://warthogs.atlassian.net/browse/WD-12825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
- shallow strip wrapping this heading pls 
image
- align button on the left on medium breakpoints
- this image needs to be 6 col wide, 16:9 aspect ratio:
image
- extra line here:
 image
- missing background here 
image
6 columns wide pls, they are too small and centred. we never centre anything:
image
- deep p-section at the obttom, always - to keep the footer further apart

**Comment by lyubomir-popov** [comment]
- image blurry, not stretching the full width of the parent container 
image
- can we pls hide the image and make the rest of this content fill the grid 
image
- these are too close. pls add some space between them:

image

- image
- can we please align this table column with the 7th column of the grid? so it lines up with content above/below:
image
- can we please do horizontal scroll in the table, and have it filling the grid on small and medium screens:
image
- same here 
image

- gap neededa bove these. remove the prule class pls
image

**Comment by immortalcodes** [comment]
- The spaces seem different here
image
image


-  Similarly here
image
image
- This seems a bit misalinged on medium screen
image
- Similarly on small screen

File: templates/azure/fips.html [hunk 1]
```diff
@@ -10,219 +10,225 @@
...
-                    alt="",
-                    width="350",
-                    height="100",
-                    hi_def=True,
-                    loading="auto",) | safe
-          }}
+        <p class="p-heading--5">For Azure compliance in production environments.</p>
+        <p>
+          Ubuntu Pro FIPS is the first and only FIPS 140-2 certified image for Azure. Built upon the enhanced stability
+          and security features of <a href="https://ubuntu.com/azure/pro">Ubuntu Pro</a>, Ubuntu FIPS is a critical
```
Comment (immortalcodes) [inline templates/azure/fips.html:25]:
```suggestion
          and security features of <a href="/azure/pro">Ubuntu Pro</a>, Ubuntu FIPS is a critical
```

File: templates/azure/fips.html [hunk 2]
```diff
@@ -10,219 +10,225 @@
...
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips"
+          class="p-button--positive"><span>Launch Ubuntu Pro FIPS 20.04 LTS</span></a>
+        <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips" style="padding-right: 1rem;"><span>18.04
+            LTS&nbsp;&rsaquo;</span></a>
+        <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial-fips"><span>16.04
+            LTS&nbsp;&rsaquo;</span></a>
+      </div>
+      <div class="col-6">
+        <div class="p-image-container--3-2 u-hide--small u-hide--medium">
+          <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/9ec8c60f-azure-heroi.png" alt="" />
```
Comment (immortalcodes) [inline templates/azure/fips.html:38]:
```suggestion
          <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/9ec8c60f-azure-heroi.png" alt="Azure logo" />
```

File: templates/azure/fips.html [hunk 3]
```diff
@@ -10,219 +10,225 @@
...
+      <h2>What's in Ubuntu Pro FIPS?</h2>
+    </div>
+    <div class="row--25-75-on-large">
+      <div class="col">
+        <hr class="is-muted u-hide--small" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <ul class="p-list">
+              <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top">FIPS compliant by default</h3>
```
Comment (immortalcodes) [inline templates/azure/fips.html:56]: [Vanilla](https://vanillaframework.io/docs/patterns/grid#nested-columns) suggests not to use pattern in nested cols

File: templates/azure/fips.html [hunk 4]
```diff
@@ -10,219 +10,225 @@
...
+              Automated auditing and remediation for the DISA STIG and CIS security benchmarks through the <a
+                href="/security/certifications/docs/usg">Ubuntu Security Guide</a>.
+            </p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--25-75-on-large">
```
Comment (immortalcodes) [inline templates/azure/fips.html:86]: Same nested cols here

File: templates/azure/fips.html [hunk 5]
```diff
@@ -10,219 +10,225 @@
...
+            </p>
+            </li>
+            </ul>
+          </div>
+          <div class="col-3 col-medium-2">
+            <ul class="p-list">
+              <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top">Predictable and Secure</h3>
+            <p>
+              Ubuntu Pro FIPS is based on <a href="https://ubuntu.com/azure/pro">Ubuntu Pro premium images</a>, with Azure
```
Comment (immortalcodes) [inline templates/azure/fips.html:106]: Please use shortened URL for all that are present on ubuntu.com

**Review by immortalcodes — CHANGES_REQUESTED** [review]
Good Work! Some minor changes

**Review by immortalcodes — APPROVED** [review]

**Pull request merged**